public-api/libproject/devstart.rb

```diff
@@ -147,33 +148,34 @@ def read_db_vars(gcc)
 
 def dev_up()
   common = Common.new
-
   account = get_auth_login_account()
   if account.nil?
     raise("Please run 'gcloud auth login' before starting the server.")
   end
 
-  at_exit { common.run_inline %W{docker-compose down} }
-  common.status "Starting database..."
-  common.run_inline %W{docker-compose up -d db}
-  common.status "Running database migrations..."
-  common.run_inline %W{docker-compose run db-migration}
-  common.run_inline %W{docker-compose run db-public-migration}
+  ServiceAccountContext.new(TEST_PROJECT).run do
```

Comment:
This should not be necessary. THis may have been from when you were having issues with ADC from the server.

Reply:
Thanks removed.